MVP/defaults-build.yaml

```diff
@@ -0,0 +1,74 @@
+- defaults:
+    name: defaults-build
+    description: |
+        Managed by Jenkins Job Builder. Do not edit via web.
+    node: 'jslave-platform-rhel7' #[MVP] please replace this value with jenkins slave name/label of your own.
+    #concurrent: true
+    parameters:
+        - string:
+            name: id
+        - string:
+            name: task_id
+        - string:
+            name: CI_TYPE
+        - string:
+            name: CI_MESSAGE
+        - string:
+            name: tag 
+        - string:
+            name: name
+        - string:
+            name: version
+        - string:
+            name: release
```

Comment:
I think they are helpful especially when our current infra systems are not stable enough.
However I'm ok with removing them for now. Getting the MVP v1 out earlier would benifit us more.